### ISSUE TITLE
IQL results different with the paper

### ISSUE THREAD

**Issue by dssrgu** [rail-berkeley/rlkit#172]
Hi,

The IQL results from this repo seem to differ from the original paper.

According to the README of the IQL example code here, IQL scores an average raw return of about 1500 on hopper-medium-expert with offline training:
https://github.com/rail-berkeley/rlkit/tree/master/examples/iql
<img width="772" alt="image" src="https://user-images.githubusercontent.com/35447981/213618310-50c4583b-1409-4b0f-9193-c8d6e4045e12.png">

However, the original paper notes that IQL scores 91.5 in normalized average return (which is about 2950 in raw return):
https://arxiv.org/pdf/2110.06169.pdf
<img width="957" alt="image" src="https://user-images.githubusercontent.com/35447981/213618835-0b811933-3985-4175-a46f-68ffb10add08.png">

Can you take a look at this and check what is causing the difference?

Thank you!

**Comment by anair13** [comment]
Sorry for the late response, but the IQL experiments in the paper were run in jax and should be reproducible with the other repo: https://github.com/ikostrikov/implicit_q_learning

This reimplementation is pytorch is for convenience and likely has minor initialization differences etc.
